### PR TITLE
Fix AnimeDao presence

### PR DESCRIPTION
**Pull request by Viruzaum**
Added a regex to get the website even if they change the url again.
Fixed video players not working.
Discord_BhVcELxdKP
Discord_EdGesLiEfg
Discord_00vbMSTKJT

### CODE REVIEW

**Comment by frozaum** [comment]
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

**Comment by Bas950** [comment]
Unlocked (No longer on hold) - Please update your fork to get your Pull Request reviewed, if you don't do it within the next 3 days the Pull Request will be closed.

**Comment by Viruzaum** [comment]
Updated and removed a console.log I forgot before

**Comment by Viruzaum** [comment]
done

File: websites/A/AnimeDao/iframe.ts [hunk 1]
```diff
@@ -3,10 +3,10 @@ const iframe = new iFrame();
 iframe.on("UpdateData", async () => {
   const { hostname } = window.location;
   if (
-    hostname === `animeproxy.info` ||
-    hostname === `rapidvid.to` ||
-    hostname === `openload.co`
+    hostname === `gounlimited.to` ||
+    hostname === `vcdn.space`
   ) {
+    console.log(hostname);
```
Comment (Bas950) [inline websites/A/AnimeDao/iframe.ts:9]: remove console log

File: websites/A/AnimeDao/presence.ts [hunk 2]
```diff
@@ -14,9 +14,9 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
-  return [Math.floor(startTime / 1000), endTime];
+  var startTime = Math.floor(Date.now()/1000);
+  var endTime = startTime - videoTime + videoDuration;
```
Comment (Bas950) [inline websites/A/AnimeDao/presence.ts:18]: const

File: websites/A/AnimeDao/presence.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ let episode,
   paused: boolean,
   played: boolean;
 
-presence.on("iFrameData", (data) => {
+presence.on("iFrameData", (data: any) => {
```
Comment (Bas950) [inline websites/A/AnimeDao/presence.ts:28]: :any isnt allowed

**Comment by Viruzaum** [comment]
done

**Comment by doomlerd** [comment]
Update your fork, it's 50 commits behind.

**Pull request closed without merging**